File: sentimental/mario/less/mario.py
from cs50 import get_int
while True:
    n = get_int("Height: ")
    if n > 0 and n <= 8:
        break
for i in range (n):
    for j in range (n-i-1):
        print(" ", end="")
    for k in range(i+1):
        print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
void print_row(int spaces, int bricks);
int main(void)
{
    // prompt user for positive integer.
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);
    // print pyramid of that height
    for (int i = 0; i < n; i++)
    {
        // print row of brick
        print_row(n- i -1,i+1);
    }
}

void print_row(int spaces, int bricks)
{ // print spaces
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }
    // print bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}